README.md

```diff
@@ -57,6 +57,52 @@ export class OtherModule {
 }
 ```
 
+### Including only what you need
+For production scenarios, you will likely only want to include the parts of ng-bootstrap that you are using in your application.
+To do so, you can import only the specific modules that you need. For example:
+
+ ```js
+ import { NgbModalModule } from '@ng-bootstrap/ng-bootstrap/modal/modal.module';
```

Comment:
I don't think this is necessary if you're using a tool like Webpack 2 or Rollup that can tree shake unused code.

The only example I can think of off the top of my head where it matters where you import from is `rxjs`, and that package is special due to how the additional operators add to the prototype of the `Observable` class.